SEUNAGBEYE/React-Redux-Es6
# #161367273 Course List Page

### What does this PR do?
Create the course list table component for displaying the list of courses in the application

### Description of Task to be completed?
- add thunk to store
- create load course thunk
- add mock api
- add course list component
- add course list row component
- create the course list table


### How should this be manually tested?
fetch this branch: `git fetch origin ft-course-list-page-161367273`
install all dependencies with `npm i`
start the app `npm start -s`
navigate to the courses page to see the table

### Any background context you want to provide?
NA


### What are the relevant pivotal tracker stories?
[#161367273](https://www.pivotaltracker.com/n/projects/2200608/stories/161367273)

### Screenshots (if appropriate)
<img width="1229" alt="screen shot 2018-10-21 at 6 41 58 pm" src="https://user-images.githubusercontent.com/25517388/47270250-05c2da00-d561-11e8-98c9-778b1c5901d0.png">
